sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -2095,7 +2183,7 @@ async def test_async_queue_send_twice(self, uamqp_transport, *, servicebus_names
     @CachedServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True)
     @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
     @ArgPasserAsync()
-    async def test_async_queue_send_timeout(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
+    async def test_async_queue_send_timeout(self, uamqp_transport, *, servicebus_namespace=None, servicebus_queue=None, **kwargs):
```

Comment:
This is the only one replacing conn str with namespace. Should this change be made everywhere?